extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLog.i18n.php

```diff
@@ -6,6 +6,7 @@
 	'discussionslog' => 'Discussions Users Log',
 	'discussionslog-desc' => 'Wikia\'s Discussion users log extension',
 	'discussionslog-pagetitle' => 'Mobile app users\' discussion logs',
+	'discussionslog-contributions-link-title' => 'discussion IP history',
```

Comment:
I was wondering about that, but decided to left it exactly as it was stated in the ticket. Will ping Cristina to clarify though.